modules/cluster-logging-updating-logging.adoc

```diff
@@ -246,12 +246,6 @@ You should see the `elasticsearch-delete-\*` and `elasticsearch-rollover-*` indi
 
 .Post update
 
-If you use Kibana, after the Elasticsearch Operator and Cluster Logging Operator are fully updated to 4.5, you must perform the following actions to complete the upgrade:
+If you use Kibana, after the Elasticsearch Operator and Cluster Logging Operator are fully updated to 4.5, you must recreate your Kibana index patterns and visualizations.
```

Comment:
I suppose the comma after Kibana in these instances is fine, so ignore my previous suggestion if you want.